docs/source/project/roadmap.rst

```diff
@@ -5,11 +5,11 @@ Roadmap
 
 This pages lists some of the features that are planned for the future of Paraglider.
 
-Features:
+Features
 ^^^^^^^^^^
 .. card::
 
-    Get the IBM plugin caught up with the other plugins in feature completeness. 
+    Get the IBM plugin caught up with the other plugins in feature completeness. [TODO: Maybe remove?]
```

Comment:
Is this TODO something you want to resolve before we merge the PR? What do we need to know to decide?